src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+### Grow your portfolio {#investing}
+
+There are fund management products on Ethereum that will try to grow your portfolio based on a strategy of your choice. This is automatic and doesn't need a human manager taking a cut of your profits.
+
+TokenSets, a popular investment product, tokenises investment strategies and calls them sets. A set represents a portfolio with a strategy. For example, their "ETH 26 Day EMA Crossover" set will aim to accumulate as much ETH as possible by rebalancing the portfolio between ETH and USDC. You don't have to manually handle any portfolio rebalancing. As long as you have the Set token in your wallet, you can withraw the portfolio as individual tokens whenever you like.
+
+### Fund your ideas {#crowdfunding}
+
+Ethereum is open source software. A lot of the work so far has been funded by the community and this has led to the growth of an interesting new fundraising model: quadratic funding. This method has the potential to improve the way we fund all types of public goods in the future.
+
+Quadratic funding makes sure that the projects that receive the most funding are those with the most unique demand. In other words, projects that stand to improve the lives of the most people. Here's how it works:
+
+1. There is a matching pool of funds donated.
+2. A round of public funding starts.
+3. People can signal their demand for a project by donating some money.
+4. Once the round is over, the matching pool is distributed to projects. Those with the most unique demand get the highest amount from the matching pool.
+
+This means project A that received 100 donations of 1 dollar could end up with more funding than project B which received a single donation of 10,000 dollars (dependent on the size of the matching pool).
+
+## How does defi work?
+
+Defi uses cryptocurrencies and smart contracts to provide services that don't need intermediaries. In today's financial world, financial institutions acts as guarantors of transactions. This gives these institutions immense power – even the power to stop you making transactions because your money flows through them. And discrimination is rife – billions of people around the world don't have access to a bank account.
+
+In defi, a smart contract replaces the financial institution in the transaction. A smart contract is a type of Ethereum account that can hold funds and can send/refund them based on certain conditions. No one can alter that smart contract when it's live. So it will always run exactly as programmed.
+
+A pocket money contract that's programmed to send money from Account A to Account B every Friday will only ever do that. So long as Account A has the required funds. No one can change the contract and add Account C as a recipient to steal funds.
+
+## Ethereum and defi
+
+Ethereum is the perfect foundation for defi for a number of reasons:
+
+- No one owns Ethereum or the smart contracts that live on it – this gives everyone an opportunity to use defi.
```

Comment:
Yes! Credible neutrality. Another consequence: the rules can't be changed on you.